src/components/ex-hero-story/ex-heroStory.scss

```diff
@@ -0,0 +1,106 @@
+.heroStory {
+  @apply mb-4;
+
+  .hero-story__content {
+    @apply flex flex-wrap-reverse;
+
+    @screen sm {
+      @apply flex-row-reverse justify-between;
+    }
+
+    @screen md {
+      @apply mt-12;
+    }
+  }
+
+  .hero-story__column {
+    &--media {
+      @apply flex-grow-0 flex-shrink-0 w-full;
+
+      @screen sm {
+        @apply w-1/2;
+      }
+    }
+
+    &--text {
+      @screen sm {
+        @apply flex flex-col flex-wrap content-start w-1/2;
+      }
+
+      * {
+        @apply max-w-full break-words;
+        hyphens: auto;
+      }
+    }
+  }
+
+  .hero-story__video {
+    @apply relative w-full overflow-hidden;
+    padding-top: 56.25%;
+
+    iframe {
+      @apply absolute top-0 left-0 w-full h-full border-none;
+    }
+  }
+
+  .hero-story__img {
+    @apply flex-grow block object-cover w-full mb-3;
+
+    @screen md {
+      @apply mb-0;
+    }
+  }
+
+  .hero-story__title {
+    @apply text-primary my-2 text-xl font-bold leading-tight;
+
+    a:hover {
+      @apply text-interactive-state-hover-link underline;
+    }
+  }
+
+  .hero-story__title-link {
+    @apply font-headline text-primary inline-block pb-2 text-xl font-semibold;
+
+    &:hover {
+      @apply text-interactive-state-hover-link underline;
+    }
+  }
+
+  .hero-story__preview {
+    @apply text-gray-0 text-default m-0;
+  }
+
+  .hero-story__topic {
+    @apply font-headline tracking-default flex items-center mt-0 mb-1 text-xs font-light leading-none text-black uppercase;
+
+    &:before {
+      @apply inline-block mr-2 bg-cover;
+
+      content: '';
+      width: 10px;
+      height: 10px;
+      background-image: url('../resources/images/sicherearbeit-arrow-right.svg');
```

Comment:
You could use the arrow from HTML entities instead of SVG: http://jsfiddle.net/L6suy/

You could inject a CSS drawn triangle using the :before element similiar to this example: https://www.30secondsofcode.org/css/s/border-with-top-triangle

Reply:
changed to html arrow